### PR TITLE
Adjusted information on Yola and added some Uralic dialects

### PR DESCRIPTION
**Pull request by d97hah**
closes #833

### CODE REVIEW

File: languoids/tree/book1242/ndon1252/md.ini [hunk 1]
```diff
@@ -44,7 +44,7 @@ multitree = njd
 [hh_ethnologue_comment]
 comment = E16/E17/E18/E19/E20/E21/E22/E23/E24 has an entry Ndonde Hamba [njd] in the Lindi region of Tanzania.  While different from its immediate neighbours,
 	it is a dialect of the language in the Makonde [kde] entry
-	(**hh:g:Kraal:Makonde**, **hh:hlde:Crosse-Upcott:KiNgindo**:13-14, 351-358) See also: Makonde [kde].
+	(**hh:g:Kraal:Makonde**, **hh:hlde:Cross-Upcott:KiNgindo**:13-14, 351-358) See also: Makonde [kde].
```
Comment (xrotwang) [inline languoids/tree/book1242/ndon1252/md.ini:47]: @d97hah I fixed this already last time, but it keeps coming back. It's *Cross**e***.

File: languoids/tree/ural1272/perm1256/komi1267/komi1268/nort3401/vymm1234/md.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+[core]
+name = Vym'
```
Comment (xrotwang) [inline languoids/tree/ural1272/perm1256/komi1267/komi1268/nort3401/vymm1234/md.ini:3]: Is the trailing single quote `'` intended?

**Review by xrotwang — CHANGES_REQUESTED** [review]

**Comment by d97hah** [comment]
Re Crosse, sorry, mea culpa!

Yes trailing ' intended in Vym'

Pada tanggal Sen, 4 Apr 2022 pukul 14.49 Robert Forkel <
***@***.***> menulis:

**Comment by kealeen** [comment]
@d97hah Hi sorry just a bit confused, will this update on the main glottolog site? Also does it say that the name is Irish Anglo-Norman instead of Yola? I just can’t tell what’s going on apologies for the inconvenience.

**Comment by xrotwang** [comment]
https://glottolog.org will display the changes once the next version is released - sometime within the the next 2 months, I'd guess.

**Comment by d97hah** [comment]
Yes, and the language level languoid will be called Irish Anglo-Norman with
two dialects Yola and Fingal. all the best, H

Pada tanggal Sel, 5 Apr 2022 pukul 20.35 Robert Forkel <
***@***.***> menulis:

**Pull request merged**